src/simple.ts

```diff
@@ -0,0 +1,32 @@
+import { defaultAbiCoder } from '@ethersproject/abi';
+import { BytesLike, HexString, toHex } from './bytes';
+import { MultiProof, processProof, processMultiProof } from './core';
+import { MerkleTreeData, MerkleTreeImpl } from './merkletree';
+import { MerkleTreeOptions } from './options';
+import { throwError } from './utils/throw-error';
+
+export function formatLeaf(value: BytesLike): HexString {
+  return defaultAbiCoder.encode(['bytes32'], [value]);
+}
+
+export class SimpleMerkleTree extends MerkleTreeImpl<BytesLike> {
+  static of = (values: BytesLike[], options: MerkleTreeOptions = {}): SimpleMerkleTree => {
+    const [tree, indexedValues] = MerkleTreeImpl.prepare(values, options, formatLeaf);
+    return new SimpleMerkleTree(tree, indexedValues, formatLeaf);
+  };
```

Comment:
I guess there's no need to keep it as a lambda:

```suggestion
  static of(values: BytesLike[], options: MerkleTreeOptions = {}): SimpleMerkleTree {
    const [tree, indexedValues] = MerkleTreeImpl.prepare(values, options, formatLeaf);
    return new SimpleMerkleTree(tree, indexedValues, formatLeaf);
  }
```